src/core/Akka.MultiNodeTestRunner/Discovery.cs

```diff
@@ -100,21 +100,21 @@ private IEnumerable<RoleName> RoleNames(Type specType)
             return roles;
         }
 
-        private ConstructorInfo FindConfigConstructor(Type configUser)
+        internal static ConstructorInfo FindConfigConstructor(Type configUser)
         {
             var baseConfigType = typeof(MultiNodeConfig);
             var current = configUser;
             while (current != null)
             {
 
 #if CORECLR
-                var ctorWithConfig = configUser
+                var ctorWithConfig = current
```

Comment:
The issue was we were never taking advantage of the loop's variable at all here. We never iterated down to the base classes effectively.